### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by Raphaeljunior**
This PR closes #10684. It adds code to drop stacktraces from non js files.

### CODE REVIEW

**Comment by Raphaeljunior** [comment]
@jridgewell PR is ready for review

File: routes/error-tracker.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const log = require('../utils/log');
 const SERVER_START_TIME = Date.now();
 const errorsToIgnore = ['stop_youtube',
   'null%20is%20not%20an%20object%20(evaluating%20%27elt.parentNode%27)'];
+const jsStackTrace = /(\.js):(\d+):(\d+)/;
```
Comment (jridgewell) [inline routes/error-tracker.js]: Nit: the group captures aren't necessary.

File: test/test-errortracker.js [hunk 2]
```diff
@@ -380,3 +380,21 @@ describe('Test stacktrace are versioned correctly', function() {
...
+  const chromeStackTraceTestInput = `at new vi (https://cdn.ampproject.org/rtv/031496877433269/v0.js:297:149)
+    at new  (https://cdn.ampproject.org/rtv/031496877433269/v0.js:298:365)
+    at dc (https://cdn.ampproject.org/rtv/031496877433269/v0.js:53:59)
+    at I (https://cdn.ampproject.org/rtv/031496877433269/v0.js:51:626)
+    at xi (https://cdn.ampproject.org/rtv/031496877433269/v0.js:298:278)
+    at mf.zc (https://cdn.ampproject.org/rtv/031496877433269/v0.js:408:166)
+    at pf (https://cdn.ampproject.org/rtv/031496877433269/v0.js:112:409)
+    at lf.$d (https://cdn.ampproject.org/rtv/031496877433269/v0.js:115:86)
+    at https://cdn.ampproject.org/rtv/031496877433269/v0.js:114:188`;
+  it('Should identify non js stacktraces', function() {
```
Comment (jridgewell) [inline test/test-errortracker.js]: We need one more test case. Some Safari stacks include `promiseReactionJob@[native code]`, which is still valid.

**Comment by Raphaeljunior** [comment]
@jridgewell I also included a fix to a bug on app engine. The version id should be a string rather than a number as previously done.

**Pull request merged**